docs/DEVELOPING.md

```diff
@@ -737,6 +737,128 @@ $ make ftest NAME=mssql
+
+### Availability and prerequisites
+
+⚠️ _Currently, this connector is available in **beta** starting in 8.8_.
+Features in beta are subject to change and are not covered by the service level agreement (SLA) of features that have reached general availability (GA).
+
+Elastic versions 8.6.0+ are compatible with Elastic connector frameworks. Your deployment must include the Elasticsearch, Kibana, and Enterprise Search services.
+
+Jira versions 7 or later are compatible with Elastic connector frameworks.
+
+### Setup and basic usage
+
+Complete the following steps to deploy the connector:
+
+1. [Gather Jira details](#gather-jira-details)
+2. [Configure Jira connector](#configure-jira-connector)
+
+#### Gather Jira details
+
+Collect the information that is required to connect to your Jira instance:
+
+- The server host url where Jira is hosted.
+- Username for the Jira server or service account for the Jira cloud.
+- Password for the Jira server or API key for the Jira cloud.
+- Project Keys to fetch the data from Jira server or cloud.
+- SSL certificate for a secure connection
+
+#### Configure Jira connector
+
+The following configuration fields need to be provided to set up the connector:
+
+##### `is_cloud`
+
+Flag to determine the Jira platform type. `True` if Jira cloud and `False` if Jira server. Default value is `True`.
+
+##### `username`
+
+The username of the account for Jira server. Default value is `admin`.
+
+##### `password`
+
+The password of the account to be used for the Jira server. Default value is `changeme`.
+
+##### `service_account_id`
+
+The service account for the Jira cloud. Default value is `me@example.com`.
+
+##### `api_token`
+
+The API key to authenticate with Jira cloud. Default value is `abc#123`.
+
+##### `host_url`
+
+The server host url where the Jira is hosted. Default value is `http://127.0.0.1:8080`. Examples:
+
+  - `https://192.158.1.38:8080/`
+  - `https://test_user.atlassian.net/`
+
+##### `projects`
+
+Comma separated `Project Keys` to fetch the data from Jira server or cloud. Default value is `*` i.e. fetches the data of all projects. Examples:
+
+  - `EC, TP`
+
+##### `ssl_enabled`
+
+Whether SSL verification will be enabled. Default value is `False`.
+
+##### `ssl_ca`
+
+Content of SSL certificate. Note: In case of ssl_enabled is `False`, keep `ssl_ca` field empty. Example certificate:
+
+  - ```
+    -----BEGIN CERTIFICATE-----
+    MIID+jCCAuKgAwIBAgIGAJJMzlxLMA0GCSqGSIb3DQEBCwUAMHoxCzAJBgNVBAYT
+    ...
+    7RhLQyWn2u00L7/9Omw=
+    -----END CERTIFICATE-----
+    ```
+
+##### `retry_count`
+
+The number of retry attempts after failed request to the Jira. Default value is `3`.
+
+##### `concurrent_downloads`
+
+The number of concurrent downloads for fetching the attachment content. This speeds up the content extraction of attachments. Defaults to `50`.
```

Comment:
I believe this was just bumped to 100 in another PR.

Reply:
Yes, let me update it.